python/arcticdb/adapters/s3_library_adapter.py

```diff
@@ -73,6 +90,11 @@ def __init__(self, uri: str, encoding_version: EncodingVersion, *args, **kwargs)
 
         self._https = uri.startswith("s3s")
         self._encoding_version = encoding_version
+        self._ca_cert_path = self._query_params.CA_cert_path
+        self._ca_cert_dir = self._query_params.CA_cert_dir
+        if (platform.system() is "Windows" or platform.system() is "Darwin") and (self._ca_cert_path or self._ca_cert_dir):
+            raise ValueError("Please use OS settings for cert management.")
```

Comment:
It will be meaningless. The sdk won't complain though.
I don't want the user to have the wrong perception that they have set the credentials in this way on Windows or Mac. A little details for user-friendliness

Reply:
It might make it difficult for people to write portable code that uses Arctic. I suppose what you have is OK for now though, we can always demote it to a log warning if people complain in practice